hybrid/reference/lattice_lnls.py

```diff
@@ -83,6 +84,10 @@ def LatticeLNLS(topology,
                    parallel with the QPU; the best result is accepted on
                    each iteration.
 
+        track_iteration_data (bool, optional, default=False):
+            Flag to indicate whether to track samples, subproblems, and subsamples (samples for the
```

Comment:
Add that the data is stored in the info field of the returned sampleset?

Reply:
added and described the correspondence between indices and num_reads